### DISCUSSION TITLE
Welcome to BezierInfo-2 Discussions!

### DISCUSSION THREAD

**Discussion by Pomax** [Pomax/BezierInfo-2#380] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ljaniec** [comment]
Hello, I mainly use Bezier curves in the mobile robot path planning tasks :) I really like the mathematical background provided here, it helps me understand how to work with them.
